.github/workflows/phpstan.yml

```diff
@@ -25,6 +25,13 @@ jobs:
         uses: actions/checkout@v4
         with:
           path: custom/plugins/${{ inputs.extensionName }}
+      - name: PHPStan cache
```

Comment:
We should do the same here by creating a `phpstan.neon` and including `phpstan.neon.dist` and set a fix
```
parameters:
  tmpDir: PATH
```

Reply:
Done